test/utils/test_unit_conversion.py

```diff
@@ -0,0 +1,65 @@
+from fpdf.util import convert_unit
+
+
+def test_convert_unit_numbers():
+    """Test converting units where both old and new are numbers"""
+    converted = convert_unit(1, 72, 72 / 25.4)
+    assert (converted - 25.4) < 0.0001  # Account for floating point errors
```

Comment:
Alternatively, you could use `math.isclose`:
https://stackoverflow.com/a/33024979/636849

Reply:
Done